cla_public/static-templates/errors/50x.html

```diff
@@ -111,7 +111,7 @@
       <section id="content" role="main">
         <div class="phase-banner">
           <p>
-            <span id="aria-feedback">Your <a href="/feedback" aria-labelledby="aria-feedback">feedback</a> will help us continue to improve this service.</span>
+            <span id="aria-feedback"><a href="https://checklegalaid.service.gov.uk/reasons-for-contacting">Report a problem</a> if you need help or would like to give feedback to improve this service</span>
```

Comment:
:raising_hand_man: We can skip the scheme and the domain if we are linking to a page in this application. This will ensure it works across all environments (development, end-to-end tests, staging, etc.)
```suggestion
            <span id="aria-feedback"><a href="/reasons-for-contacting">Report a problem</a> if you need help or would like to give feedback to improve this service</span>
```